components/forms/Login/index.jsx

```diff
@@ -13,35 +20,83 @@ export const loginFormTestIds = {
 };
 
 const LoginForm = () => {
+  const { mutateUser } = useUser({
+    redirectTo: '/',
+    redirectIfFound: true,
+  });
+
+  const SUCCESS_TITLE = 'Login failed';
+  const SUCCESS_DESCRIPTION = 'Check whether you email is registered with us or if your password is correct';
```

Comment:
```suggestion
  const UNSUCCESS_TITLE = 'Login failed';
  const UNSUCCESS_DESCRIPTION = 'Check whether you email is registered with us or if your password is correct';
```

I'm not sure if I suggested it wrongly last time I review this const naming 🤔  But as these consts describe an unsuccessfull login event, they should be named accordingly ;-)

Reply:
done in d001d237faad2946d3f13cd8a0e2207485664da4